frontend/public/components/graphs/prometheus-graph.tsx

```diff
@@ -49,7 +49,7 @@ export const PrometheusGraphLink_: React.FC<PrometheusGraphLinkProps> = ({
       : `/dev-monitoring/ns/${namespace}/metrics?${params.toString()}`;
 
   return (
-    <Link to={url} style={{ color: 'inherit', textDecoration: 'none' }}>
+    <Link to={url} aria-label={url} style={{ color: 'inherit', textDecoration: 'none' }}>
```

Comment:
This should be some text describing the link, right? Like "View Graph in Query Browser"

cc @jessiehuff

Reply:
Exactly, it would be some text to describe to screen reader users what the link is. (This [article](https://www.washington.edu/accessibility/links/) explains meaningful link text fairly well.) Having the url itself can be confusing to screen reader users so I think Sam's suggestion would be better.

Reply:
ok, yeah, wasn't sure of this fix.  Thanks for the link, I'll read and update.

Reply:
Hi, first line of article states "Link text should be unique within a page", so added title of charts to the `Link's aria-label`:

![image](https://user-images.githubusercontent.com/12733153/95753475-ab504600-0c6f-11eb-908c-7ebd5bf0937d.png)